README.md

```diff
@@ -45,8 +45,8 @@ required python libraries on your own. See [requirements.txt](requirements.txt)
 
 ```shell
 usage: oelint-adv [-h] [--suppress SUPPRESS] [--output OUTPUT] [--fix] [--nobackup] [--addrules ADDRULES [ADDRULES ...]]
-                  [--customrules CUSTOMRULES [CUSTOMRULES ...]] [--rulefile RULEFILE] [--jobs JOBS] [--color] [--quiet] [--noinfo]
-                  [--nowarn] [--relpaths] [--noid] [--messageformat MESSAGEFORMAT] [--constantmods CONSTANTMODS [CONSTANTMODS ...]] [--print-rulefile] [--exit-zero]
```

Comment:
I kind of get the motivation of the PR, *but* removing a CLI flag is a breaking change, thus it would require a major version bump, which I would like to avoid for now.

So for me the following proposal would be a way forward:
- keep --nowarn, --noinfo, but add argparse.SUPPRESS to the configuration, to have them disapprearing from the help
- automatically map the nowarn and noinfo values into the new hide parameter in `core.arguments_post`
- remove the mention from the README

Reply:
That's quite the good point, I will re add nowarn and noinfo to keep compatibility but keep it hiden from help and README